entity/contents.h

```diff
@@ -119,16 +121,26 @@ class SolidStrokeContents final : public Contents {
     kButt,
     kRound,
     kSquare,
-    kLast,
   };
 
   enum class Join {
     kMiter,
     kRound,
     kBevel,
-    kLast,
   };
 
+  typedef std::function<void(
```

Comment:
Please prefer `using` instead of `typedef` unless the TU is meant to be C.

Reply:
Done.